src/pages/docs/chat/index.mdx

```diff
@@ -9,13 +9,72 @@ Ably Chat is a product dedicated to making it quick and easy to build chat funct
 
 The Chat SDK contains a set of purpose-built APIs that abstract away the complexities involved in how you would architect chat features. It utilizes Ably's platform to benefit from all of the same performance guarantees and scaling potential.
 
-* [Getting started: Chat in JavaScript / TypeScript](/docs/chat/getting-started/javascript)
-* [Getting started: Chat in React](/docs/chat/getting-started/react)
-* [Getting started: Chat UI Kits in React](/docs/chat/getting-started/react-ui-components)
-* [Getting started: Chat in React Native](/docs/chat/getting-started/react-native)
-* [Getting started: Chat in Android](/docs/chat/getting-started/android)
-* [Getting started: Chat in Kotlin (JVM)](/docs/chat/getting-started/jvm)
-* [Getting started: Chat in Swift](/docs/chat/getting-started/swift)
+## Why Chat
+
+### Built for Serious Scale & Reliability
+
+Ably Chat is powered by Ably's globally distributed platform that handles **550 billion messages per month** and **1 billion+ connections per month**, serving the world's most demanding workloads with **100% uptime over 5+ years**. With a global edge network featuring **635 points of presence**, Ably Chat delivers messages with **&lt;50ms median roundtrip latency** and provides automatic multi-region failover to ensure your chat experiences are always available.
```

Comment:
Might be worth splitting the scale part and reliability part into separate paragraphs here, otherwise you get a a lot of 'withs' together, and a lot of bold text. Might also need some revision with an updated section above that covers it being built on the platform. 

We should also link out to the platform section, or architectural section when discussing these numbers. 

Should '550 billion' have a + ?